### PR TITLE
Replace simple textarea by code editor

### PR DESCRIPTION
**Pull request by LisaSuperDudu**
We used codemirror to include JavaScript syntax highlighting and line numbers.
PTAL :) :) <3 <3 #endOfDay

### CODE REVIEW

**Comment by jonathanperret** [comment]
Thanks for this PR! A better editor will surely improve the Paysage experience.

I don't see a dependency on `codemirror` being declared in `bower.json` though, and indeed when I run your code I get a 404 on `codemirror.js` and associated files. Could you please fix this?

File: views/programmer.hbs [hunk 1]
```diff
@@ -51,5 +56,12 @@
 <script>
 $(Paysage.programmerInit);
 </script>
+<script>
```
Comment (jonathanperret) [inline views/programmer.hbs]: I'd like to keep a single `<script>` tag here if possible. It looks like this code could be moved into the `Paysage.programmerInit` function, keeping all initialization together.

**Comment by jonathanperret** [comment]
I just tested the result, looks awesome! ❤️ 

I just commented about the initialization code, if you can tighten it a bit I'll be delighted to merge this. Here's a rabbit: 🐇

**Comment by LisaSuperDudu** [comment]
Glad you like it! Here's a magic hat 🎩

**Comment by etienneCharignon** [comment]
Attention, apparently the code is not reloaded when you click on the name of a creature in the liste of creature in the developper view.

**Comment by etienneCharignon** [comment]
after some more testing, it appears that there is the two following layout problems 
- on firefox : the scrolling does not work
- on chrome : when entering a very long line (without space) the preview pane disappear.

**Comment by jonathanperret** [comment]
Merging, thanks! 🌈🦄

**Pull request merged**